### PR TITLE
[jenkins]: Add sonic-stp build job

### PR DESCRIPTION
**Pull request by rupesh-k**

### CODE REVIEW

File: scripts/common/sonic-stp-build/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -xe
```
Comment (xumia) [inline scripts/common/sonic-stp-build/build.sh:1]: Please change the file permission for the file, make it executable.
-rw-rw-r--
Change to
-rwxrwxr-x
See: https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-stp-build/2/console
Permission denied

Comment (rupesh-k) [reply scripts/common/sonic-stp-build/build.sh:1]: Done. taken care.

File: scripts/common/sonic-stp-build/build.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+#install libevent_2
+sudo apt-get -y purge libevent-dev
+sudo dpkg -i target/debs/stretch/libevent_2.*.deb
```
Comment (xumia) [inline scripts/common/sonic-stp-build/build.sh:5]: The file does not exist, see: https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-all/lastSuccessfulBuild/artifact/target/debs/stretch/

Comment (ph408077) [reply scripts/common/sonic-stp-build/build.sh:5]: Libevent version 2.1.8 was added as part of STP feature and the same will be available once below PR is merged.
https://github.com/Azure/sonic-buildimage/pull/3463/

**Pull request merged**